scripts/restart-server

```diff
@@ -97,6 +97,7 @@ if has_application_server():
                 [
                     "zulip_deliver_scheduled_emails",
                     "zulip_deliver_scheduled_messages",
+                    "zulip_clear_user_status",
```

Comment:
I think `zulip_clear_expired_user_statuses` would feel like a more consistent and clearer name for this job.